multicluster/charts/linkerd-multicluster/README.md

```diff
@@ -80,7 +80,7 @@ Kubernetes: `>=1.21.0-0`
 | gateway.port | int | `4143` | The port on which all the gateway will accept incoming traffic |
 | gateway.probe.path | string | `"/ready"` | The path that will be used by remote clusters for determining whether the gateway is alive |
 | gateway.probe.port | int | `4191` | The port used for liveliness probing |
-| gateway.probe.seconds | int | `3` |  |
+| gateway.probe.seconds | int | `3` | The interval (in seconds) between liveness probes |
```

Comment:
weird.... why were documentation missing on certain values?  Was this a bug in helm-docs 1.4.0?

Reply:
Yeah, something like that.